wallet-new/test/unit/WalletUnitTest.hs

```diff
@@ -25,17 +25,21 @@ import           TxMetaStorageSpecs (txMetaStorageSpecs)
 main :: IO ()
 main = do
 --    _showContext
-    runTranslateNoErrors $ withConfig $ return $ hspec tests
+    -- runTranslateNoErrors $ withConfig $ return $ hspec tests
+
+    -- TODO mhueschen | asses whether this is ok. the above machinery seems
+    -- unnecessary
+    hspec tests
```

Comment:
It would also be good to have input from the `wallet-new` team on this. I'm unclear on what `runTranslateNoErrors $ withConfig $ return $` gets us. Since I've [modified `runTranslateNoErrors`](https://github.com/input-output-hk/cardano-sl/pull/3559/commits/91db28b92416fc5d1d9ef3d96836715effd615fe#diff-5f43ed2f6b541395cadae109a207d773R122), the piping of consistent configuration becomes difficult/impossible at this toplevel location.

cc/ @adinapoli-iohk @edsko @matt-noonan

Reply:
I think this is a remnant of the past. `tests` already has a type that is independent from any constraints; I don't think this is doing anything useful.

Reply:
Thanks @edsko